pages/workspace/workspace.ejs

```diff
@@ -84,17 +90,25 @@
           <span class="nav-item badge badge-light"><%= navTitle %></span>
         </li>
         <li class="nav-item ml-2 my-1">
-          <a id="reboot" class="nav-item btn btn-light" data-toggle="modal" data-target="#rebootModal">
+          <button id="reboot" class="nav-item btn btn-light" data-toggle="modal" data-target="#rebootModal">
             <i class="fas fa-sync text-info" aria-hidden="true"></i>
             Reboot
-          </a>
+          </button>
         </li>
         <li class="nav-item ml-2 my-1">
-          <a id="reset" class="nav-item btn btn-light" data-toggle="modal" data-target="#resetModal">
+          <button id="reset" class="nav-item btn btn-light" data-toggle="modal" data-target="#resetModal">
             <i class="fas fa-trash text-danger" aria-hidden="true"></i>
             Reset
-          </a>
+          </button>
         </li>
+        <% if (showLogs) { %>
+          <li class="nav-item ml-2 my-1">
+            <a class="nav-item btn btn-light" href="<%= urlPrefix %>/workspace/<%= workspace_id %>/logs" target="_blank">
+              <i class="fas fa-arrow-up-right-from-square" aria-hidden="true"></i>
```

Comment:
Any idea for a better icon? I couldn't find anything that screamed "logs" to me 🤔

Reply:
Maybe https://fontawesome.com/icons/bars-staggered?s=solid&f=classic?

Reply:
I like `bars-staggered`!

Reply:
Changed in 8c73f53.